src/FSharpPlus/Extensions/Seq.fs

```diff
@@ -78,6 +78,18 @@ module Seq =
                     members <- ResizeArray ()
                     members.Add e.Current
             yield g, members }
+    
+    /// <summary>
```

Comment:
Should we include `zipShortest` even in types like `seq` and `neseq` which already have `zip` with that behavior?

I think this is not necessary.

Reply:
True, removed